packages/sirius-web/backend/sirius-web/src/test/resources/scripts/migration.sql

```diff
@@ -64,6 +77,24 @@ INSERT INTO document (
          "id": "9674f8f7-ff1a-4061-bb32-a4a235a9c2ca",
          "eClass": "view:View",
          "data": {
+           "colorPalettes": [
+             {
+               "data": {
+                 "colors": [
+                   {
+                     "data": {
+                       "name": "color_empty",
+                       "value": ""
+                     },
+                     "eClass": "view:FixedColor",
+                     "id": "63184ddc-74c4-4888-bb65-418361689e2b"
+                   }
+                 ]
+               },
+               "eClass": "view:ColorPalette",
+               "id": "d315989f-826f-490d-b898-d94200b0caa2"
+             }
+           ],
```

Comment:
Do not modify the test data for `NodeDescription#labelExpression migration`, create your own test data. It won't be possible to have one test for all possible migrations